### ISSUE TITLE
Autosomal Dominant with genetic anticipation

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7486]
**Preferred term label:**
Autosomal Dominant with genetic anticipation

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of autosomal dominant inheritance involving a gene that exhibits anticipation, the increase in severity and/or an earlier age of onset in subsequent generations.

**Parent term (use hpo.jax.org/app)**
Autosomal dominant inheritance HP:0000006

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts I have updated the definition and placed the term under the new Modifier hierarchy.
The label 'Genetic anticipation' seemed better than 'Autosomal Dominant with genetic anticipation' since databases will need to use bot 'Autosomal dominant' and 'Genetic anticipation' as annotations, and so I added Autosomal Dominant with genetic anticipation as a synonym. Please reopen if you disagree.
